src/main/io/osd.c

```diff
@@ -645,19 +645,22 @@ static bool osdDrawSingleElement(uint8_t item)
 
     case OSD_WARNINGS:
         {
+
+#define OSD_WARNINGS_MAX_SIZE 11
+
```

Comment:
Maybe worth defining OSD_FORMAT_MESSAGE_BUFFER_SIZE as `OSD_WARNINGS_MAX_SIZE+1` (or similar?), to prevent repeating?

Reply:
Good point, done, thanks.